.github/workflows/renv.yml

```diff
@@ -47,18 +47,24 @@ jobs:
       matrix:
         # Run all steps in this job for the following repositories.
         repos:
-          - name: admiralonco
-            target-branch: devel
-          - name: admiraldev
-            target-branch: devel
-          - name: admiraltemplate
-            target-branch: main
-          - name: admiralophtha
-            target-branch: devel
+          # - name: admiralonco
+          #   target-branch: devel
+          # - name: admiraldev
+          #   target-branch: devel
+          # - name: admiraltemplate
+          #   target-branch: main
```

Comment:
can this get removed unless there is a reason for keeping it that i am not aware of??

Reply:
Definitely, this is unfortunately still work in progress.

I wanted to test the workflow on one repo only, but since automatic adding of reviewers works as it should, I'll extend that to all other repos and uncomment this part.

I'll finish that tomorrow, together with regenerating the `renv.lock` based on `devel` versions of Admiral packages.

Reply:
And for `admiraltemplate` in particular - of course I can remove that if not required.

Reply:
sorry didn't realize that was commented out and its purpose!  apologies.  

so the `admiraltemplate` repo should have the `renv.lock` file propagated to it, but it shouldn't be put into the `renv.lock` file as it is just a template repo for setting up an R package.  Apologies this stuff is mysterious to me!!